README.md

```diff
@@ -100,6 +100,9 @@ Again we plot measurement:
     1.  `cd ~/performance_ws`
     1.  `wget https://github.com/ros2/buildfarm_perf_tests/raw/master/tools/ros2_dependencies.repos`
     1.  `vcs import src < ros2_dependencies.repos`
+1.  Install required packages:
+    1. `sudo pip3 install pandas matplotlib`
+    1. `sudo apt-get install libboost-all-dev`
```

Comment:
The dependencies should be specified in the `package.xml` and installed through by rosdep so that the instructions are not specific to ubuntu and don't need to be updated when the dependencies change.

Also, I'm not sure why the boost dependency is necessary.

```suggestion
    1. `rosdep install --from-path src --ignore-src`
```

Reply:
Thank you for comment.

> The dependencies should be specified in the package.xml

Thank you, I misunderstand. We can write non ROS packages dependencies in  `packages.xml`!

>  the boost dependency is necessary

performance_test needs it, please see below.
But as it is specified in `src/performance_test/performance_test/package.xml`,  `rosdep install` looks to fix everything.

Without boost, I got the following error.

```
$ colcon build --symlink-install
Starting >>> performance_test
--- stderr: performance_test                           
CMake Error at /usr/share/cmake-3.16/Modules/FindPackageHandleStandardArgs.cmake:146 (message):
  Could NOT find Boost (missing: Boost_INCLUDE_DIR program_options timer)
Call Stack (most recent call first):
  /usr/share/cmake-3.16/Modules/FindPackageHandleStandardArgs.cmake:393 (_FPHSA_FAILURE_MESSAGE)
  /usr/share/cmake-3.16/Modules/FindBoost.cmake:2179 (find_package_handle_standard_args)
  CMakeLists.txt:27 (find_package)

---
Failed   <<< performance_test [14.2s, exited with code 1] 

Summary: 0 packages finished [15.0s]
  1 package failed: performance_test
  1 package had stderr output: performance_test
  1 package not processed
```